FamilySearchIosSampleApp/AncestorDetails.swift

```diff
@@ -36,68 +36,71 @@ class AncestorDetails : UIViewController
+                                            self?.ancestorBirthLabelValue.hidden = false
                                             
                                             // death data
                                             if (personDetails?.personDeathDate != nil)
                                             {
-                                                self.ancestorDeathLabelTitle.text = NSLocalizedString("ancestorDetailsDeath", comment: "Death:")
-                                                self.ancestorDeathLabelTitle.hidden = false
-                                                self.ancestorDeathLabelValue.text = personDetails?.personDeathDate
-                                                self.ancestorDeathLabelValue.hidden = false
+                                                self?.ancestorDeathLabelTitle.text = NSLocalizedString("ancestorDetailsDeath", comment: "Death:")
+                                                self?.ancestorDeathLabelTitle.hidden = false
+                                                self?.ancestorDeathLabelValue.text = personDetails?.personDeathDate
+                                                self?.ancestorDeathLabelValue.hidden = false
                                             }
                                         })
                                     }
                                 })
     }
     
-    func getAncestorDetailsData(personUrlString:String,
+    func getAncestorDetailsData(personUrlString:String?,
                                 accessToken:String,
                                 completionAncestorDetails:(responseDetails: PersonDetails?, reponseError:NSError?) -> ())
     {
+        let ancestorDetailsUrl = NSURL(string: personUrlString!);
+
         let configuration = NSURLSessionConfiguration.defaultSessionConfiguration();
         let headers: [NSObject : AnyObject] = ["Accept":"application/json", "Authorization":"Bearer " + accessToken];
         configuration.HTTPAdditionalHeaders = headers;
         let session = NSURLSession(configuration: configuration)
-        let ancestorDetailDataTask = session.dataTaskWithURL(NSURL(string: personUrlString)!) { (ancestorData, ancestorResponse, ancestorError) in
+        let ancestorDetailDataTask = session.dataTaskWithURL(ancestorDetailsUrl!) { (ancestorData, ancestorResponse, ancestorError) in
             if (ancestorError == nil)
             {
                 do
                 {
                     let ancestryDataJson = try NSJSONSerialization.JSONObjectWithData(ancestorData!, options: .AllowFragments);
                     // print("ancestryDataJson = \(ancestryDataJson)")
 
-                    let persons = ancestryDataJson["persons"] as! [[String:AnyObject]]
-                    let person = persons.first
+                    let persons = ancestryDataJson["persons"] as? [[String:AnyObject]]
```

Comment:
There are a lot of assumptions being made in how the data will be populated in this section.  All of the force unwrapping is really unsafe.  I can only think of one case where force unwrapping an optional is sort of okay and that's when I do a null check in a conditional and then force unwrap in the same conditional to check something.  Swift doesn't handle that need very well yet.  Any other case (I can think of) we should really be unwrapping optionals with either a `guard` of `if-let`